Neos.Neos/Classes/FrontendRouting/Projection/DocumentUriPathFinder.php

```diff
@@ -195,6 +195,28 @@ public function getLastChildNode(
         );
     }
 
+    /**
+     * @throws NodeNotFoundException
+     * @internal
+     */
+    public function getLastChildNodeNotBeing(
```

Comment:
naming was a little hard to grasp but when looking at the parameter `$excludedNodeAggregateId` it _should_ explain.